src/gui/tray/SyncStatus.qml

```diff
@@ -0,0 +1,115 @@
+import QtQuick 2.15
+import QtQuick.Controls 2.15
+import QtQuick.Layouts 1.15
+
+import Style 1.0
+
+import com.nextcloud.desktopclient 1.0 as NC
+
+Item {
+    id: item
+
+    property var model: NC.SyncStatusModel {}
+
+    implicitHeight: layout.height
+
+    RowLayout {
+        id: layout
+
+        Layout.alignment: Qt.AlignLeft
```

Comment:
you are inside an `Item` this line will have no effect
you will only have an effect when using attached properties like this one when you are a direct child of the type (in this case `Layout` is a type inherited by `RowLayout`)

Reply:
thanks. was a left over.